environments/staging/inventory.ini

```diff
@@ -64,16 +64,16 @@ couch3
 # Amazon EC2
 rabbit2
 
-[kafka1]
-10.201.40.14 hostname=kafka1-staging ufw_private_interface=ens5 ansible_python_interpreter=/usr/bin/python3 datavol_device=/dev/sdf datavol_device1=/dev/sdf is_datavol_ebsnvme=yes root_encryption_mode=aws kafka_broker_id=0 datavol_fstype=xfs
+[kafka2]
+10.201.40.170 hostname=kafka2-staging ufw_private_interface=ens5 ansible_python_interpreter=/usr/bin/python3 datavol_device=/dev/sdf datavol_device1=/dev/sdf is_datavol_ebsnvme=yes root_encryption_mode=aws kafka_broker_id=1
```

Comment:
How come `datavol_fstype=xfs` is set for couch and formplayer, but it's not set for kafka any more?

Reply:
those are formatted as xfs type and all new servers are formatting as ext4 now